From 30ad88cdcd43732e519a7cca87c359e615aa6b9f Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Mon, 18 Mar 2024 17:15:25 +0100
Subject: [PATCH] arm64: dts: st: enable Ethernet1 on stm32mp257f-ev1 board

Enable Ethernet instance (ETHERNET1) which is put in standalone config.
(connected to Realtek PHY in RGMII mode)

Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Change-Id: I0bf32a2a85c813d71debde80dbd8aebe7c9b2414
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/367532
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 993368a5e4be..41206a1509f4 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -237,7 +237,7 @@ dsi_out1: endpoint {
 };
 
 &eth1 {
-	status = "disabled";
+	status = "okay";
 	pinctrl-0 = <&eth1_rgmii_pins_a &eth1_mdio_pins_a>;
 	pinctrl-1 = <&eth1_rgmii_sleep_pins_a &eth1_mdio_sleep_pins_a>;
 	pinctrl-names = "default", "sleep";
-- 
2.39.5

